gravity-forms/gw-cache-buster.php

```diff
@@ -271,6 +272,25 @@ public function filter_resume_link( $resume_url, $form, $resume_token, $email )
 		return add_query_arg( array( 'gf_token' => $resume_token ), $referer );
 	}
 
+	public function replace_embed_url( $text, $form, $entry, $url_encode ) {
+		// Check if the text contains the {embed_url} merge tag
+		if ( strpos( $text, '{embed_url}' ) !== false ) {
+	
+			$url        = $_SERVER['REQUEST_URI'];
+			$parsed_url = parse_url($url);
+	
+			parse_str( $parsed_url['query'], $query_params );
+	
+			$form_request_origin = rgar( $query_params, 'form_request_origin' );
+			$decoded_value       = urldecode($form_request_origin);
+			$original_url        = $_SERVER['HTTP_ORIGIN'] . $decoded_value;
```

Comment:
Can this all be simplified to the following?

```suggestion
			$original_url = rgget( 'form_request_origin' );
```

Reply:
@saifsultanc Sorry, this was a bad suggestion. I didn't realize that `form_request_origin` was `$_SERVER['REQUEST_URI']`.

```php
$form_request_origin = rgar( $query_params, 'form_request_origin' );
$decoded_value         = urldecode( $form_request_origin );
$origin_url                   = $_SERVER['HTTP_ORIGIN'] . $decoded_value;
```

... does make sense here, otherwise it'll just be the path.